oneshot.d/browser-update-default-data

```diff
@@ -8,8 +8,7 @@ fi
 
 if [ ! -f "$MOZEMBED_DIR/ua-update.json" -o ! -s "$MOZEMBED_DIR/ua-update.json" ]; then
     # Remove lines starting with a comment.
-    # Using Perl regexp so that this works on device.
-    grep -vP  ^\\s*// $BROWSER_DATA_DIR/ua-update.json.in > "$MOZEMBED_DIR/ua-update.json"
+    grep -vE  '^\s*//' $BROWSER_DATA_DIR/ua-update.json.in > "$MOZEMBED_DIR/ua-update.json"
```

Comment:
Apparently requires updated grep if done like proposed. Already with existing setup could done like this to get the same result:

`sed  '/^\s*\/\// d' $BROWSER_DATA_DIR/ua-update.json.in > "$MOZEMBED_DIR/ua-update.json"`

Reply:
This sed solution looks good. I didn't happen to test this grep -E on a device that has GNU grep, although that would have been good, and basically just assumed that if it works on busybox it works with GNU as well. Have to be more careful next time.

Btw, is the title fine? I'm wondering if it should be [sailfish-browser] instead of [oneshot] to look good on change log. It might be that it doesn't really matter.

Reply:
It's just name of the component that is in the square brackets. Technically doesn't matter.
Title is good.

Reply:
Approved also this way.